babel.config.js

```diff
@@ -1,31 +1,23 @@
+const { NODE_ENV } = process.env;
+
 module.exports = {
   presets: [
+    '@babel/typescript',
     [
-      '@babel/preset-env',
+      '@babel/env',
       {
-        useBuiltIns: 'false',
-        corejs: 'core-js@3',
         targets: {
-          chrome: 68,
+          browsers: ['ie >= 11'],
         },
-      },
-    ],
-    '@babel/preset-typescript',
-  ],
-  plugins: [
-    ['@babel/plugin-proposal-decorators', { legacy: true }],
-    [
-      '@babel/plugin-proposal-class-properties',
-      {
+        exclude: ['transform-async-to-generator', 'transform-regenerator'],
+        modules: false,
         loose: true,
       },
     ],
-    '@babel/plugin-syntax-dynamic-import',
-  ],
-  overrides: [
-    {
-      test: /\.ts$/,
-      plugins: [['@babel/plugin-proposal-decorators', { legacy: true }]],
-    },
   ],
+  plugins: [
+    // don't use `loose` mode here - need to copy symbols when spreading
+    '@babel/proposal-object-rest-spread',
+    NODE_ENV === 'test' && '@babel/transform-modules-commonjs',
```

Comment:
Why only apply on test mode. 
Why use @babel/transform-modules-commonjs?

Reply:
Refer to the configuration of babel in the [rollup document](https://rollupjs.org/guide/en/#babel)
>Before Babel will actually compile your code, it needs to be configured. Create a new file, src/.babelrc:
```
{
  "presets": [
    ["@babel/env", {"modules": false}]
  ]
}
```
>There are a few unusual elements to this setup. First, we're setting "modules": false, otherwise Babel will convert our modules to CommonJS before Rollup gets a chance to do its thing, causing it to fail.

When we test prompt error:
```
  ● Test suite failed to run

    /Users/Anzhu/Work/dicomloader/src/calc/add.test.ts:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import { add } from './add';
                                                                                             ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:579:25)
```

So add `'@babel/transform-modules-commonjs'` when testing.